.github/workflows/validate.yaml

```diff
@@ -134,7 +134,7 @@ jobs:
         run: |
           helm repo add bitnami https://charts.bitnami.com/bitnami
           helm repo update
-          helm install redis bitnami/redis
+          helm install redis bitnami/redis --version 17.14.5
```

Comment:
Just curious, why do we want to pin?  this can catch us later on too when we forget :)

Reply:
Per Arturs comment above ^, tho also not sure on the exact reasoning for it